util/util.go

```diff
@@ -32,15 +32,23 @@ func DeriveKey(secret, salt []byte, length uint) ([]byte, error) {
 	return key, nil
 }
 
-// MustGetenv returns the environment variable `name` if it exists or panics otherwise
-func MustGetenv(name string) string {
-	value := os.Getenv(name)
-	if len(value) == 0 {
+// MustGetEnv returns the environment variable `name` if it exists or panics otherwise
+func MustGetEnv(name string) string {
```

Comment:
```suggestion
// MustGetenv returns the environment variable `name` if it exists or panics otherwise
func MustGetenv(name string) string {
```
suggest naming the functions MustGetenv, Getenv, and MustGetwd in accordance with os package